glean-core/rlb/src/test.rs

```diff
@@ -1369,6 +1369,31 @@ fn test_boolean_get_num_errors() {
     assert_eq!(result, 0);
 }
 
+#[test]
+fn test_text_can_hold_long_string() {
+    let _lock = lock_test();
+
+    let _t = new_glean(None, false);
+
+    let metric = TextMetric::new(CommonMetricData {
+        name: "text_metric".into(),
+        category: "test".into(),
+        send_in_pings: vec!["custom1".into()],
+        lifetime: Lifetime::Application,
+        disabled: false,
+        dynamic_label: Some(str::to_string("text")),
+    });
+
+    // 216 characters, which would overflow StringMetric
+    metric.set(str::to_string("I've seen things you people wouldn't believe. Attack ships on fire off the shoulder of Orion. I watched C-beams glitter in the dark near the Tannhäuser Gate. All those moments will be lost in time, like tears in rain"));
```

Comment:
Movie quotes are very acceptable, especially ones with non-ASCII characters in them.

Idiomatic Rust might be `"I've seen things...rain".into()` instead of `str::to_string("I've seen things...rain")`

Reply:
updated 👍